libs/stats/odc/stats/_gm_ls_bitmask.py

```diff
@@ -148,14 +160,14 @@ def reduce(self, xx: xr.Dataset) -> xr.Dataset:
         gm = gm.rename(self.renames)
 
         # Scale USGS Landsat bands into surface reflectance
```

Comment:
It's really only scaling to the range 0-10000 (approx). It's already surface reflectance. This is a pedantic note though!